core/CronArchive/SegmentArchiving.php

```diff
@@ -294,7 +305,7 @@ public function getAllSegmentsToArchive($idSite)
         return $segments;
     }
 
-    private function isSegmentForSite($segment, $idSite)
+    public function isSegmentForSite($segment, $idSite)
```

Comment:
Any reason for having that on public now? Seems it isn't used outside the class. Not even in tests...